### PR TITLE
bugfix.

### PR DESCRIPTION
**Pull request by maksimfedin**

### CODE REVIEW

File: service-node/src/main/scala/org/enterprisedlt/fabric/service/node/cryptography/FabricCryptoMaterial.scala [hunk 1]
```diff
@@ -296,7 +292,7 @@ object FabricCryptoMaterial {
     ): CertAndKey = {
         CryptoUtil.createSignedCert(
             OrgMeta(
-                name = s"$componentName.$organization",
+                name = s"$componentName",
```
Comment (apolubelov) [inline service-node/src/main/scala/org/enterprisedlt/fabric/service/node/cryptography/FabricCryptoMaterial.scala]: just `componentName`

File: service-node/src/main/scala/org/enterprisedlt/fabric/service/node/cryptography/FabricCryptoMaterial.scala [hunk 2]
```diff
@@ -324,7 +319,7 @@ object FabricCryptoMaterial {
...
```
Comment (apolubelov) [inline service-node/src/main/scala/org/enterprisedlt/fabric/service/node/cryptography/FabricCryptoMaterial.scala]: just `componentName`

File: service-node/src/main/scala/org/enterprisedlt/fabric/service/node/cryptography/FabricCryptoMaterial.scala [hunk 3]
```diff
@@ -337,7 +332,7 @@ object FabricCryptoMaterial {
                 UseForEncipherment,
                 UseForClientAuth,
                 UseForServerAuth,
-                AlternativeDNSName(s"$componentName.$organization"),
+                AlternativeDNSName(s"$componentName"),
```
Comment (apolubelov) [inline service-node/src/main/scala/org/enterprisedlt/fabric/service/node/cryptography/FabricCryptoMaterial.scala]: just remove, it's literally the same as next line

**Review by apolubelov — CHANGES_REQUESTED** [review]

File: service-node/src/main/scala/org/enterprisedlt/fabric/service/node/cryptography/FabricCryptoMaterial.scala (hunk 2 above)
Comment (maksimfedin) [reply service-node/src/main/scala/org/enterprisedlt/fabric/service/node/cryptography/FabricCryptoMaterial.scala]: fixed.

File: service-node/src/main/scala/org/enterprisedlt/fabric/service/node/cryptography/FabricCryptoMaterial.scala (hunk 3 above)
Comment (maksimfedin) [reply service-node/src/main/scala/org/enterprisedlt/fabric/service/node/cryptography/FabricCryptoMaterial.scala]: fixed.

**Comment by maksimfedin** [comment]
pls consider to merge.

**Review by apolubelov — APPROVED** [review]

**Pull request merged**